apps/user-group-cdc/src/main/java/it/pagopa/selfcare/user/event/mapper/UserGroupNotificationMapper.java

```diff
@@ -3,9 +3,11 @@
 import it.pagopa.selfcare.user.event.entity.UserGroupEntity;
 import it.pagopa.selfcare.user.model.UserGroupNotificationToSend;
 import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
 
 @Mapper(componentModel = "cdi")
 public interface UserGroupNotificationMapper {
 
+    @Mapping(target = "id", expression = "java(userGroupEntity.getId().toString())")
```

Comment:
While toString() on ObjectId typically returns the hexadecimal representation, it's safer and clearer to use toHexString() to ensure consistency....In addition, ensure that userGroupEntity.getId() is not null to avoid NullPointerException. You can add a default mapping or use conditional mapping if necessary.